docs/babel.config.js

```diff
@@ -55,13 +34,6 @@ module.exports = {
     'babel-plugin-optimize-clsx',
     // for IE11 support
     '@babel/plugin-transform-object-assign',
-    [
```

Comment:
It now uses the TypeScript aliases 

https://nextjs.org/docs/advanced-features/module-path-aliases